universal-application-tool-0.0.1/app/assets/javascripts/main.ts

```diff
@@ -54,6 +54,20 @@ function maybeShowWarning() {
   }
 }
 
+ /** If a toast message is present, make it visible for the specified duration. */
+function maybeShowToast() {
+  const toastMessage = document.querySelector('.cf-toast');
+  if (toastMessage) {
+    toastMessage.classList.remove('opacity-0');
+    if (toastMessage.hasAttribute('duration')) {
+      const showDuration = toastMessage.getAttribute('duration');
+      setTimeout(function() {
+        toastMessage.classList.add('opacity-0');

```

Comment:
Issue (blocking): will this cause there to be an invisible element on the page that could cover and prevent interaction with other UI?

Reply:
Yes, it's possible. I'll fix that now along with the multiple toast portion (and convert the demo warning to a dismissible toast while I'm at it.)